solr/core/src/test/org/apache/solr/cloud/TestRandomFlRTGCloud.java

```diff
@@ -70,6 +70,9 @@
 
   /** Always included in fl so we can vet what doc we're looking at */
   private static final FlValidator ID_VALIDATOR = new SimpleFieldValueValidator("id");
+
+  /** Since nested documents are not tested, when _root_ is declared in schema, it is always the same as id */
+  private static final FlValidator ROOT_VALIDATOR = new RenameFieldValueValidator("id" , "_root_");
```

Comment:
Added a validator for _root_, which is now added automatically since the schema used here declares _root_.